docs/reviewing.md

```diff
@@ -113,3 +113,30 @@ When components are defined in the `package_info` in `conanfile.py` the followin
 
 - use the `cmake_find_package` or `cmake_find_package_multi` generators in `test_package/conanfile.py`
 - corresponding call to `find_package()` with the components _explicitly_ used in `target_link_libraries`
+
+### Recommended feature options names
+
+It's often needed to add options to toggle specific library features on/off. Sometimes, it's preferred to have options enabled by default, sometimes disabled by default. In order to avoid the fragmentation, we recommend to use the following naming conventions for such options:
+
+- enable_xxx / disable_xxx
```

Comment:
Typically this is in regards to "features"

I think there's a split on `enable_feature` and just `feature`